tests/src/test/scala/com/nvidia/spark/rapids/GpuExpressionTestSuite.scala

```diff
@@ -172,6 +172,11 @@ abstract class GpuExpressionTestSuite extends SparkQueryCompareTestSuite {
     }
   }
 
+  def compareStringifiedDecimals(expected: String, actual: String): Boolean = {
+    (expected == null && actual == null) ||
+        (expected != null && actual != null && Decimal(expected) == Decimal(actual))
```

Comment:
I thought we wanted to produce the exact same result.  If we don't produce the exact same result then at a minimum we need to document the incompatibility and if it is a big enough difference, then we need to have a config to disable it by default.